components/EvergageABTest.tsx

```diff
@@ -56,11 +60,14 @@ export default class EvergageABTest extends React.Component<IEvergageABTestProps
         const chosenExperience = indexOf(this.props.variants, experience.variant, function(inArr, variant) {
             return inArr.name === variant;
         }); 
-        if(chosenExperience != null){
+        if(chosenExperience != -1){
```

Comment:
-1 is returned when an element is not found similar to the indexOf array method.

Reply:
is the .includes() method available?

Reply:
or maybe give this an identifier